core/rawdb/fileutil.go

```diff
@@ -19,12 +19,13 @@
 
 package rawdb
 
-import "github.com/prometheus/tsdb/fileutil"
+import "github.com/gofrs/flock"
```

Comment:
this LGTM, but @PlasmaPower should also review

Reply:
LGTM -- longer term we might want to move the FileLock interface to a common file between this and `filelock_mock.go` but no need to make that change now